source/PlayerInfo.cpp

```diff
@@ -1101,9 +1104,15 @@ map<const shared_ptr<Ship>, vector<string>> PlayerInfo::FlightCheck() const
 			if(ship->CanBeCarried() || !ship->HasBays())
 				continue;
 
-			for(auto &bay : ship->Bays())
+			auto &bays = ship->Bays();
+			totalBays += bays.size();
+			for(auto &bay : bays)
 			{
-				++bayCount[bay.category];
+				if(bay.bayType)
+					for(const string &category : bay.bayType->Categories())
+						++bayCount[category];
+				else
```

Comment:
this looks confusing, I would actually prefer the {} here

Reply:
```suggestion
				if(bay.bayType)
				{
					for(const string &category : bay.bayType->Categories())
						++bayCount[category];
				}
				else
```